### PR TITLE
Graphical Overmap

### PR DESCRIPTION
**Pull request by Larwck**
#### Summary
SUMMARY: Mods "Adds Graphical Overmap mod"

#### Purpose of change
Current overmap is hard to read for some.

#### Describe the solution
Using a bitmap font as a base, I created some icons for various objects on the overmap. I then created a JSON that modifies the symbols (and colors) of each overmap object to reference the graphics I made, which I placed arbitrarily over glyphs that would be rarely if ever used (apologies to Europe).

#### Describe alternatives you've considered
Full tile support for the overmap window - way out of my ability

#### Testing
Used a save where I had access to a large overmap to test areas. Some locations I could not test because of this, including necropolis and various underground or above ground-level areas. Most likely locations for bugs (where a letter is used instead of an icon) are in those locations.

#### Additional context
graphical overmap screenshot
Any opinions on color changes appreciated.
Some locations are hidden in forests. However, some of these have the symbol and color of a forest, while their name reveals what they really are (eg. "cabin_aban1"). Not sure whether to make them look like forests or as their name suggests. So far have made them the symbol of their name, but green.

### CODE REVIEW

File: data/mods/Graphical_Overmap/fontlist.txt [hunk 1]
```diff
@@ -0,0 +1,2796 @@
+Map Font LARWICK
```
Comment (ZhilkinSerg) [inline data/mods/Graphical_Overmap/fontlist.txt]: You don't need to add this file to a mod - it is generated on end systems automatically.

Comment (Larwck) [reply data/mods/Graphical_Overmap/fontlist.txt]: Ahah, that makes a lot of sense, thanks!

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/using-bitmap-as-a-font/22107/2

**Comment by evankimori** [comment]
Having an issue using GO alongside some other mods in the latest PR build. Should I create an issue for this or wait a little bit to let the mod settle in?

Don't wanna spam issue submissions for GO's compatibility with other mods if the author is already aware.

**Comment by ZhilkinSerg** [comment]
> 
> 
> Having an issue using GO alongside some other mods in the latest PR build. Should I create an issue for this or wait a little bit to let the mod settle in?
> 
> Don't wanna spam issue submissions for GO's compatibility with other mods if the author is already aware.

Try here - https://github.com/CleverRaven/Cataclysm-DDA/pull/35792

**Pull request merged**